mail_compose_select_lang/models/report.py

```diff
@@ -0,0 +1,18 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+# Copyright 2015 Pedro M. Baeza
+# For license notices, see __openerp__.py file in root directory
+##############################################################################
+from openerp import models, api
+
+
+class Report(models.Model):
+    _inherit = 'report'
+
+    @api.model
+    def translate_doc(self, doc_id, model, lang_field, template, values):
```

Comment:
As the change is only done over a context parameter, and this parameter is only affected when another one parameter is used, it's very unlinkely that there's any side effect on this code. A test would be good, but this one in special is complicated to be done and I don't have the time for now.